include/zephyr/bluetooth/direction.h

```diff
@@ -359,4 +359,26 @@ int bt_df_conn_cte_rsp_enable(struct bt_conn *conn);
  */
 int bt_df_conn_cte_rsp_disable(struct bt_conn *conn);
 
+/**
+ * @brief Convert CTE type value from HCI format to @ref bt_df_cte_type format.
+ *
+ * @param hci_cte_type   CTE type in an HCI format.
+ *
+ * @return CTE type (@ref bt_df_cte_type).
+ */
+static inline int bt_df_get_bt_df_cte_type(uint8_t hci_cte_type)
+{
+	switch (hci_cte_type) {
+	case BT_HCI_LE_AOA_CTE:
+		return BT_DF_CTE_TYPE_AOA;
+	case BT_HCI_LE_AOD_CTE_1US:
+		return BT_DF_CTE_TYPE_AOD_1US;
+	case BT_HCI_LE_AOD_CTE_2US:
+		return BT_DF_CTE_TYPE_AOD_2US;
+	case BT_HCI_LE_NO_CTE:
+		return BT_DF_CTE_TYPE_NONE;
+	default:
+		return BT_DF_CTE_TYPE_NONE;
+	}
+}
```

Comment:
Given that it's unlikely that an application would use this, I would perhaps put it in `direction_internal.h` instead.

Reply:
Hmm, actually, since it's also used in by other files (such as scan.c), perhaps place it next to `bt_get_phy` which does the same thing, but for PHYs. 

For something else (but somewhat related), we seemingly have both `BT_DF_CTE_TYPE_AOA` and `BT_GAP_CTE_AOA` (and duplicates of the others as well). 

Doesn't need to be fixed in this PR, but ideally we should only define those values one place.

Reply:
Thank you, I moved it to direction_internal.h

Reply:
I moved a conversion function to hci_core next to bt_get_phy()